### PR TITLE
Multi-tag (intersection) support.

### PR DESCRIPTION
**Pull request by steve-bate**
In a tagged view, the tags include a + (add) and - (remove) control to modify the set of tags being queried. In a single-tag view, the current tag will have no controls. Also supports multi-tag RSS feeds.

### CODE REVIEW

File: src/routes/core.js
Comment (j12i) [inline src/routes/core.js:123]: is this safe? Say I'd put `; DROP TABLE bookmarks;` as the second tag. Would this be caught anywhere?

Comment (steve-bate) [reply src/routes/core.js:123]: It's a good question. The example you gave wouldn't work, but others might have. I modified the queries to use parameters. I've run `sqlmap` against the URL and it doesn't find any vulnerabilities.

**Comment by ckolderup** [comment]
ah, shoot, can I apply this merge request to the PR somehow when it's coming from a fork I don't have commit access to? 🤔

**Comment by steve-bate** [comment]
I don't know. Is this a conflict from the eslint integration? I can't take a look at fixing the conflicts on my end.

**Comment by ckolderup** [comment]
it is, yeah. I tried generating a patchfile that I *think* might work? https://gist.github.com/ckolderup/f8cdaa9ee0742183c8cb95fc239cbae6

**Comment by ckolderup** [comment]
hmm, no, maybe not. Can't `git am` with a clean copy of the PR.

**Comment by steve-bate** [comment]
I just force-pushed a clean PR with the merge conflicts addressed (and squashed).

**Comment by ckolderup** [comment]
that works, thanks. I'm seeing a few random eslint warnings but it'll be easier to just address those after merging.

**Review by ckolderup — APPROVED** [review]
thanks for putting this together! Closes #55.

**Comment by steve-bate** [comment]
Thanks. I'll be sure to run eslint on any future PRs.

**Pull request merged**
